flake.nix

```diff
@@ -1,13 +1,13 @@
 {
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/master";
+    nixpkgs.url = "github:yihuang/nixpkgs/master";
     flake-utils.url = "github:numtide/flake-utils";
     nix-bundle-exe = {
       url = "github:3noch/nix-bundle-exe";
       flake = false;
     };
     gomod2nix = {
-      url = "github:nix-community/gomod2nix";
+      url = "github:yihuang/gomod2nix";
```

Comment:
https://github.com/nix-community/gomod2nix/pull/107

Reply:
workaround by specify go explicitly in `default.nix`, without use the forked repo.